konduit-serving-core/pom.xml

```diff
@@ -278,6 +278,12 @@
             <artifactId>logback-classic</artifactId>
             <version>${logback.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>uk.org.lidalia</groupId>
+            <artifactId>sysout-over-slf4j</artifactId>
+            <version>1.0.2</version>
+        </dependency>
```

Comment:
I wasn't aware of this, it's cool.
MIT/X11 license should be fine for us too.
Let's put version into parent pom properties.